package.json

```diff
@@ -1,21 +1,26 @@
 {
   "name": "set-value",
+  "version": "3.0.1",
   "description": "Create nested values and any intermediaries using dot notation (`'a.b.c'`) paths.",
-  "version": "3.0.2",
```

Comment:
Looks like the version should be updated to the latest... I'm sure this will be bumped before publishing, but it might cause confusion when looking at the source on GitHub.

Reply:
that's not the latest? I pulled down. Damn I wonder how I missed that.